examples/optimization_sequential/README.md

```diff
@@ -2,15 +2,18 @@
 
 This simple example illustrates how Phoenics can be used for the sequential optimization of a single objective function. We consider the Branin function for this illustration, which is non-convex with three global minima. 
 
-### Configuration details
+## Configuration details
 
 Configuration details for this optimization procedure are specified in `config.json`
 
 ```json
 {
     "general": {
-        "parallel": true,
-        "boosted":  false
+		"batches": 1,
+		"parallel": true,
+		"boosted":  true,
+		"backend": "edward",
```

Comment:
We should use tensorflow probability as a default backend

Reply:
Replaced `edward` with `tfprob`.